### PR TITLE
Enforce rubocop on more files

### PR DESCRIPTION
**Pull request by teceler**
Now covered:
* Seed files
* Most of the config files (but not the ones where we're monkey patching)
* Some bin files

### CODE REVIEW

File: bin/yarn [hunk 1]
```diff
@@ -4,8 +4,8 @@ Dir.chdir(APP_ROOT) do
   begin
     exec "yarnpkg", *ARGV
   rescue Errno::ENOENT
-    $stderr.puts "Yarn executable was not detected in the system."
-    $stderr.puts "Download Yarn at https://yarnpkg.com/en/docs/install"
+    warn "Yarn executable was not detected in the system."
+    warn "Download Yarn at https://yarnpkg.com/en/docs/install"
```
Comment (Throne3d) [inline bin/yarn:8]: Weird, I think this is Rails default

Comment (teceler) [reply bin/yarn:8]: I can revert it and tell rubocop to ignore it if we object, but I figured it wasn't a big deal given we don't... use... yarn... anyway...

Comment (Throne3d) [reply bin/yarn:8]: Probably OK, we just might end up reverting it with the next `rails app:update`

**Review by Throne3d — APPROVED** [review]
(Needs a rebase but) I forgot to approve this last time! Approved

**Pull request merged**